<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoveComp</class>
 <widget class="QDialog" name="LoveComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>781</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/lovecomp.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lovecheck">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>23</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>CHECK!</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_capricorn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Capricorn</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_pisces">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Pisces</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_gemini">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Gemini</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_aquarius">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Aquarius</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_aries">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>82</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Aries</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_taurus">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Taurus</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cancer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Cancer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_leo">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Leo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_saggitarius">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Saggitarius</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_libra">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Libra</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_virgo">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Virgo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_scorpio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:darkblue</string>
   </property>
   <property name="text">
    <string>Scorpio</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_lovecheck</sender>
   <signal>clicked()</signal>
   <receiver>LoveComp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
